frontend/src/scenes/sceneLogic.ts

```diff
@@ -15,7 +15,11 @@ import { SceneExport, Params, Scene, SceneConfig, SceneParams, LoadedScene } fro
 import { emptySceneParams, preloadedScenes, redirects, routes, sceneConfigurations, scenes } from 'scenes/scenes'
 import { FEATURE_FLAGS } from 'lib/constants'
 
+// Don't import and use this logic directly as via "scenes/scenes" it pulls in all scenes with
+// their dynamic imports, and hence confuses bundlers.
+// Instead use sceneProxyLogic.
```

Comment:
Nothing will crash, and no bundle will grow out of control. What happened for me was `eventUsageLogic` was linked from every one of our 3 bundles (app, toolbar, shared), and since it linked to sceneLogic, all the 3 bundles reloaded on any change to any file. If that happens again, it'll be quick to spot and fix.

I reworked this now, and pass the `scenes`  to `sceneLogic` via `props`, not directly through an `import`. This ties all the scenes to `App.tsx` instead, which isn't really imported from anywhere.